.github/workflow-scripts/gh-pages/get-pr-data.js

```diff
@@ -0,0 +1,15 @@
+module.exports = async ({
+  pullNumber,
+  github,
+  context: { repo: { repo, owner } },
+}) => {
+  pullNumber = parseInt(pullNumber.replace('#', ''));
```

Comment:
Why would the pullNumber have a `#` in it?

Reply:
Because it's an input on the workflow so someone might add the `#`.

Reply:
Okie doke :+1: